src/pages/Contests/Contests.jsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState, useEffect } from 'react'
+// import Loading from '../logreg/loading.jsx'
+// import Navbar from '../../components/Header/Navbar'
+// import FooterSmall from '../../components/Footer/FooterSmall'
+import queryString from 'query-string'
+import update from 'immutability-helper' // it'll be used in the func to change the difficulty. Immutability-helper is an alternative for react-addon-update (react-addon-update is no longer maintained).
+import AccordionCom from '../../components/AccordionCom'
+import difficultyObject from '../../utils/StaticData'
+import ConstestAPI from '../../actions/Contest'
+
+import 'antd/dist/antd.css'
```

Comment:
Try adding this import in App.js file and check if it is being accessed in this file.